packages/server/src/modules/app/index.ts

```diff
@@ -86,6 +86,9 @@ application.test.start = async () => {
   await connect();
   // TODO - should be a part of the lifecycle hook/event
   app.use(application.middleware.errorHandler);
+
+  const request = await import('./tests/request');
```

Comment:
I don't understand this part. What's the purpose of it?

Reply:
Seems like a leftover, fixed